modules/options/src/main/scala/scala/build/actionable/ActionableDiagnostic.scala

```diff
@@ -0,0 +1,38 @@
+package scala.build.actionable
+
+import dependency._
+
+import scala.build.Position
+import scala.build.errors.{Diagnostic, Severity}
+
+sealed abstract class ActionableDiagnostic {
```

Comment:
I'd say this one doesn't need to be sealed (we shouldn't need to match `ActionableDiagnostic` on all their possible instance types). That should allow to move `ActionableDependencyUpdateDiagnostic` alongside its handler.